README.md

```diff
@@ -178,3 +178,27 @@ and then refactor your code from `GwtEvent` to `Event`).
    import org.gwtproject.event.legacy.shared.EventHandler;
    import org.gwtproject.event.legacy.shared.GwtEvent;
    ```
+
+## Instructions
+
+To build gwt-event:
+
+* run `mvn clean install`
+
+on the parent directory.
+
+To run the j2cl tests:
+
+* switch to the 'gwt-timer-j2cl-tests' directory
+* run `mvn j2cl:clean` & `mvn j2cl:test`
```

Comment:
This was added to tell people how to run the j2cl tests. Inside the latest README we already removed it.

Reply:
Yep, no longer needed. Before htmlunit support it was preferred, since it was manual.

The `j2cl:clean` is the rough equivelent of your gwt-m-p's writing gwt-unitCache outside of target, so it needs an explicit clean operation, not bound by default - was never needed, but like most clean+install, just get cargo culted in after it fixes a problem once

Reply:
Removed